.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build_and_test:
     docker:
-      - image: circleci/node:6.10
+      - image: cumuluss/circleci:node-6.10
```

Comment:
Should this be `cumulus` rather than `cumuluss`?

Reply:
ok, just looked at docker hub and the answer is no, `cumuluss` is not a typo but dang it looks like one. Is that meant to be a temporary account?

Reply:
unfortunately, `cumulus` namespace is taken on docker. We are using `cumuluss` for now :(

Reply:
Wouldn't multiple cumulus items be "cumuli"?

Reply:
@yjpa7145 great point. @sethvincent do we have `cumuli`?